pyscriptjs/src/components/pyconfig.ts

```diff
@@ -59,6 +113,27 @@ export class PyConfig extends BaseEvalElement {
         this.remove();
     }
 
+    loadEnv = async () => {
```

Comment:
if we are using packages in the config maybe rename to `loadPackages` to match nomenclature?

Reply:
done